rest/examples.md

```diff
@@ -1,20 +1,3 @@
 # Examples
-Sample APIs following the guidelines are available at [adidas-group GitHub](https://github.com/adidas-group). The samples include the following:
```

Comment:
Same, we should provide some examples, it's pity that we remove them :)

Reply:
I think that is a low hanging fruit, right? We can use our workshop API plus I guess @zdne has some which were used as templates/samples before? 

If it just adds time, maybe lets put this action (examples) in the backlog and go forward though?

Reply:
@Amzani @jerzyn the guidelines still mentions the examples in this GitHub repo. The "good old" Order example is still there. Marking as resolved for this PR.